tests/attacks/inference/test_attribute_inference.py

```diff
@@ -0,0 +1,211 @@
+# MIT License
+#
+# Copyright (C) IBM Corporation 2020
```

Comment:
Could you please change this line to
```
# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2020
```
This is the author/copyright convention we recently adopted.